src/phpDocumentor/Plugin/Core/ServiceProvider.php

```diff
@@ -24,102 +25,91 @@
  * phpDocumentor and, for backwards compatibility, registers the service providers for Graphs, Twig and PDF to
  * the container.
  */
-final class ServiceProvider implements ServiceProviderInterface
+final class ServiceProvider
```

Comment:
I hope is one is on the refactor wish list? Way to much properties in here,  and I don't understand what this class is doing.